Makefile

```diff
@@ -2,13 +2,23 @@
 # https://github.com/precice/calculix-adapter/wiki/Installation-instructions-for-CalculiX
 # Set the following variables before building:
 # Path to original CalculiX source (e.g. $(HOME)/ccx_2.16/src )
-CCX             = $(HOME)/PathTo/CalculiX/ccx_2.16/src
-# Path to SPOOLES main directory (e.g. $(HOME)/SPOOLES.2.2 )
-SPOOLES         = $(HOME)/PathTo/SPOOLES
-# Path to ARPACK main directory (e.g. $(HOME)/ARPACK )
-ARPACK          = $(HOME)/PathTo/ARPACK
-# Path to yaml-cpp prefix (e.g. $(HOME)/yaml-cpp, should contain "include" and "build")
-YAML            = $(HOME)/PathTo/yaml-cpp
+CCX             = $(HOME)/CalculiX/ccx_2.16/src
+
+### Change these if you built SPOOLES, ARPACK, or yaml-cpp from source ###
+# SPOOLES include directory (e.g. $(HOME)/SPOOLES.2.2 )
+SPOOLES_INCLUDE   = /usr/include/spooles/
+# SPOOLES library flags (e.g. $(HOME)/SPOOLES.2.2/spooles.a)
+SPOOLES_LIBS      = -L/usr/lib/x86_64-linux-gnu/ -lspooles
+#
+# ARPACK include directory (e.g. $(HOME)/ARPACK)
+# ARPACK_INCLUDE  =
+# ARPACK library flags (e.g. $(HOME)/ARPACK/libarpack_INTEL.a)
+ARPACK_LIBS       = -L/usr/lib/ -larpack -llapack -lblas
+#
+# yaml-cpp include path (e.g. $(HOME)/yaml-cpp/include)
+YAML_INCLUDE      = /usr/include/
+# yaml-cpp library flags (e.g. -L$(HOME)/yaml-cpp/build -lyaml-cpp)
+YAML_LIBS         = -L/usr/lib/x86_64-linux-gnu/ -lyaml-cpp
```

Comment:
Are the `-L` flags necessary here? If the linker cannot find these libraries out of the box then there will be runtime errors as their locations are unknown. (If they are shared libraries)

Reply:
They are not necessary here, but I am still providing them explicitly as an example for the user. But indeed, removing them would be more robust. I will remove them.